flang/test/Integration/OpenMP/atomic-capture-complex.f90

```diff
@@ -13,16 +13,16 @@
 !CHECK: %[[VAL_1:.*]] = alloca { float, float }, i64 1, align 8
```

Comment:
TODO: Add new test(s) to reproduce the bug fixed by the PR once the reviewers agree with the taken approach.